--- LPC82x.svd.orig	2017-12-06 07:34:02.208608070 +0100
+++ LPC82x.svd	2017-12-06 07:52:04.777364108 +0100
@@ -641,9 +641,29 @@
 							</enumeratedValues>
 						</field>
 						<field>
+							<name>SEL_EXTCLK</name>
+							<description>Select external or internal clock source for the self-wake-up timer. The internal
+clock source is selected by the CLKSEL bit in this register if SET_EXTCLK is set
+to internal.</description>
+							<bitRange>[3:3]</bitRange>
+							<enumeratedValues>
+								<name>ENUM</name>
+								<enumeratedValue>
+									<name>INTERNAL</name>
+									<description>Internal. The clock source is the internal clock selected by the CLKSEL bit.</description>
+									<value>0</value>
+								</enumeratedValue>
+								<enumeratedValue>
+									<name>EXTERNAL</name>
+									<description>External. The self-wake-up timer uses the external WKTCLKIN pin.</description>
+									<value>1</value>
+								</enumeratedValue>
+							</enumeratedValues>
+						</field>
+						<field>
 							<name>RESERVED</name>
 							<description>Reserved.</description>
-							<bitRange>[31:3]</bitRange>
+							<bitRange>[31:4]</bitRange>
 							
 						</field>
 					</fields>
