--- LPC82x.svd.orig	2017-12-06 06:48:44.796592325 +0100
+++ LPC82x.svd	2017-12-06 07:05:20.434004357 +0100
@@ -272,14 +272,9 @@
 			<field>			
 				<name>IVALUE</name>		
 				<description>Time interval load value. This value is loaded into the TIMERn register and the MRTn starts counting down from IVALUE -1. If the timer is idle, writing a non-zero value to this bit field starts the timer immediately.  If the timer is running, writing a zero to this bit field does the following: If LOAD = 1, the timer stops immediately. If LOAD = 0, the timer stops at the end of the time interval.</description>		
-				<bitRange>[23:0]</bitRange>		
+				<bitRange>[30:0]</bitRange>		
 					
 			</field>
-			<field>
-			<name>RESERVED</name>
-			<description>Reserved, user software should not write ones to reserved bits. The value read from a reserved bit is not defined.</description>
-			<bitRange>[30:24]</bitRange>	
-			</field>
 			<field>			
 				<name>LOAD</name>		
 				<description>Determines how the timer interval value IVALUE is loaded into the TIMERn register. This bit is write-only. Reading this bit always returns 0.</description>		
@@ -314,12 +309,12 @@
 			<field>			
 				<name>VALUE</name>		
 				<description>Holds the current timer value of the down-counter. The initial value is loaded as IVALUE - 1 from the TIME_INTVALn register either at the end of the time interval if the LOAD bit in TIME_INTVALn is 0 and the timer is in repeat mode or immediately if LOAD = 1. When the timer is in idle state, reading this bit fields returns -1 (0x7FFF FFFF).</description>		
-				<bitRange>[23:0]</bitRange>		
+				<bitRange>[30:0]</bitRange>		
 			</field>			
 			<field>			
 				<name>RESERVED</name>		
 				<description>Reserved.</description>		
-				<bitRange>[31:24]</bitRange>		
+				<bitRange>[31:31]</bitRange>		
 			</field>			
 		</fields>				
 	</register>					
