--- LPC82x.svd	2017-07-14 12:21:33.053443434 +0200
+++ LPC82x.fixed.svd	2017-07-14 12:13:04.192320201 +0200
@@ -9700,12 +9700,12 @@
 				<enumeratedValues>												
 				<name>ENUM</name>												
 					<enumeratedValue>											
-						<name>1__20_MHZ_FREQUENCY</name>										
+						<name>1_TO_20_MHZ_FREQUENCY</name>										
 						<description>1 - 20 MHz frequency range.</description>										
 						<value>0</value>										
 					</enumeratedValue>											
 					<enumeratedValue>											
-						<name>15__25_MHZ_FREQUENC</name>										
+						<name>15_TO_25_MHZ_FREQUENC</name>										
 						<description>15 - 25 MHz frequency range</description>										
 						<value>1</value>										
 					</enumeratedValue>											
@@ -17304,12 +17304,12 @@
 					<enumeratedValues>
 					<name>ENUM</name>
 						<enumeratedValue>
-							<name>ACTIVE_LOW__FALLING</name>
+							<name>ACTIVE_LOW_FALLING</name>
 							<description>Active low - falling edge. Hardware trigger is active low or falling edge triggered, based on TRIGTYPE.</description>
 							<value>0</value>
 						</enumeratedValue>
 						<enumeratedValue>
-							<name>ACTIVE_HIGH__RISING</name>
+							<name>ACTIVE_HIGH_RISING</name>
 							<description>Active high - rising edge. Hardware trigger is active high or rising edge triggered, based on TRIGTYPE.</description>
 							<value>1</value>
 						</enumeratedValue>
