--- LPC82x.svd.orig	2017-12-06 07:08:08.657469952 +0100
+++ LPC82x.svd	2017-12-06 07:32:37.735215197 +0100
@@ -9578,9 +9578,55 @@
 			<field>													
 				<name>RESERVED</name>												
 				<description>Reserved</description>												
-				<bitRange>[31:17]</bitRange>												
+				<bitRange>[23:17]</bitRange>												
 												
 			</field>													
+			<field>
+				<name>ADC_RST_N</name>
+				<description>ADC reset control</description>
+				<bitRange>[24:24]</bitRange>
+				<enumeratedValues>
+				<name>ENUM</name>
+					<enumeratedValue>
+						<name>ASSERT_THE_ADC_RESET</name>
+						<description>Assert the ADC reset.</description>
+						<value>0</value>
+					</enumeratedValue>
+					<enumeratedValue>
+						<name>CLEAR_THE_ADC_RESET</name>
+						<description>Clear the ADC reset.</description>
+						<value>1</value>
+					</enumeratedValue>
+				</enumeratedValues>
+			</field>
+			<field>
+				<name>RESERVED</name>
+				<description>Reserved</description>
+				<bitRange>[28:25]</bitRange>
+			</field>
+			<field>
+				<name>DMA_RST_N</name>
+				<description>DMA reset control</description>
+				<bitRange>[29:29]</bitRange>
+				<enumeratedValues>
+				<name>ENUM</name>
+					<enumeratedValue>
+						<name>ASSERT_THE_DMA_RESET</name>
+						<description>Assert the DMA reset.</description>
+						<value>0</value>
+					</enumeratedValue>
+					<enumeratedValue>
+						<name>CLEAR_THE_DMA_RESET</name>
+						<description>Clear the DMA reset.</description>
+						<value>1</value>
+					</enumeratedValue>
+				</enumeratedValues>
+			</field>
+			<field>
+				<name>RESERVED</name>
+				<description>Reserved</description>
+				<bitRange>[31:30]</bitRange>
+			</field>
 		</fields>														
 	</register>															
 	<register>															
