--- LPC82x.svd.orig	2017-08-23 12:22:23.755409053 +0200
+++ LPC82x.svd	2017-08-23 12:22:27.222081960 +0200
@@ -17733,7 +17733,7 @@
 					<description>Byte pin registers port 0; pins PIO0_0 to PIO0_28</description>
 			
 					<addressOffset>0x0000</addressOffset>
-					<size>1</size>
+					<size>8</size>
 					<access>read-write</access>											
 					<resetValue>0</resetValue>											
 					<resetMask>0x00000000</resetMask>	
