--- LPC845.svd.orig	2019-03-07 20:55:52.732144676 +0100
+++ LPC845.svd	2019-03-07 22:00:56.263335056 +0100
@@ -16429,7 +16439,7 @@
                   <value>0x1</value>
                 </enumeratedValue>
                 <enumeratedValue>
-                  <name>FRO0CLK</name>
+                  <name>FRG0CLK</name>
                   <description>Frg0clk</description>
                   <value>0x2</value>
                 </enumeratedValue>
