--- LPC845.svd.orig	2019-03-06 13:53:04.768231577 +0100
+++ LPC845.svd	2019-03-06 13:54:46.089814612 +0100
@@ -400,31 +400,6 @@
           </register>
         </cluster>
         <register>
-          <name>MODCFG</name>
-          <description>Module Configuration register. This register provides information about this particular MRT instance.</description>
-          <addressOffset>0xF0</addressOffset>
-          <size>32</size>
-          <access>read-write</access>
-          <resetValue>0x1F4</resetValue>
-          <resetMask>0x1FF</resetMask>
-          <fields>
-            <field>
-              <name>NOC</name>
-              <description>Identifies the number of channels in this MRT.(4 channels on this device.)</description>
-              <bitOffset>0</bitOffset>
-              <bitWidth>4</bitWidth>
-              <access>read-only</access>
-            </field>
-            <field>
-              <name>NOB</name>
-              <description>Identifies the number of timer bits in this MRT. (31 bits wide on this device.)</description>
-              <bitOffset>4</bitOffset>
-              <bitWidth>5</bitWidth>
-              <access>read-only</access>
-            </field>
-          </fields>
-        </register>
-        <register>
           <name>IDLE_CH</name>
           <description>Idle channel register. This register returns the number of the first idle channel.</description>
           <addressOffset>0xF4</addressOffset>
