--- LPC845.svd.orig	2019-03-07 19:30:01.328706630 +0100
+++ LPC845.svd	2019-03-07 20:21:33.705958618 +0100
@@ -21046,44 +21046,6 @@
               </enumeratedValues>
             </field>
             <field>
-              <name>MODE32K</name>
-              <description>Selects standard or 32 kHz clocking mode.</description>
-              <bitOffset>7</bitOffset>
-              <bitWidth>1</bitWidth>
-              <access>read-write</access>
-              <enumeratedValues>
-                <enumeratedValue>
-                  <name>DISABLED</name>
-                  <description>Disabled. USART uses standard clocking.</description>
-                  <value>0</value>
-                </enumeratedValue>
-                <enumeratedValue>
-                  <name>ENABLED</name>
-                  <description>Enabled. USART uses the 32 kHz clock from the RTC oscillator as the clock source to the BRG, and uses a special bit clocking scheme.</description>
-                  <value>0x1</value>
-                </enumeratedValue>
-              </enumeratedValues>
-            </field>
-            <field>
-              <name>LINMODE</name>
-              <description>LIN break mode enable.</description>
-              <bitOffset>8</bitOffset>
-              <bitWidth>1</bitWidth>
-              <access>read-write</access>
-              <enumeratedValues>
-                <enumeratedValue>
-                  <name>DISABLED</name>
-                  <description>Disabled. Break detect and generate is configured for normal operation.</description>
-                  <value>0</value>
-                </enumeratedValue>
-                <enumeratedValue>
-                  <name>ENABLED</name>
-                  <description>Enabled. Break detect and generate is configured for LIN bus operation.</description>
-                  <value>0x1</value>
-                </enumeratedValue>
-              </enumeratedValues>
-            </field>
-            <field>
               <name>CTSEN</name>
               <description>CTS Enable. Determines whether CTS is used for flow control. CTS can be from the input pin, or from the USART's own RTS if loopback mode is enabled.</description>
               <bitOffset>9</bitOffset>
