--- LPC845.svd.orig	2019-03-03 22:18:14.000000000 +0100
+++ LPC845.svd	2019-03-06 12:48:09.023986047 +0100
@@ -239,7 +247,7 @@
             <size>32</size>
             <access>read-write</access>
             <resetValue>0</resetValue>
-            <resetMask>0x80FFFFFF</resetMask>
+            <resetMask>0xFFFFFFFF</resetMask>
             <fields>
               <field>
                 <name>IVALUE</name>
@@ -275,8 +283,8 @@
             <addressOffset>0x4</addressOffset>
             <size>32</size>
             <access>read-only</access>
-            <resetValue>0xFFFFFF</resetValue>
-            <resetMask>0xFFFFFF</resetMask>
+            <resetValue>0x00FFFFFF</resetValue>
+            <resetMask>0xFFFFFFFF</resetMask>
             <fields>
               <field>
                 <name>VALUE</name>
