--- LPC845.svd.orig	2019-03-06 13:47:56.146086315 +0100
+++ LPC845.svd	2019-03-06 13:49:44.438081131 +0100
@@ -33,6 +33,10 @@
         <size>0x1C</size>
         <usage>registers</usage>
       </addressBlock>
+      <interrupt>
+        <name>WDT</name>
+        <value>12</value>
+      </interrupt>
       <registers>
         <register>
           <name>MOD</name>
@@ -225,6 +229,10 @@
         <size>0xFC</size>
         <usage>registers</usage>
       </addressBlock>
+      <interrupt>
+        <name>MRT</name>
+        <value>10</value>
+      </interrupt>
       <registers>
         <cluster>
           <dim>4</dim>
@@ -3992,6 +4000,10 @@
         <size>0x8</size>
         <usage>registers</usage>
       </addressBlock>
+      <interrupt>
+        <name>CMP</name>
+        <value>11</value>
+      </interrupt>
       <registers>
         <register>
           <name>CTRL</name>
@@ -5158,6 +5170,10 @@
         <size>0xFEC</size>
         <usage>registers</usage>
       </addressBlock>
+      <interrupt>
+        <name>FLASH</name>
+        <value>14</value>
+      </interrupt>
       <registers>
         <register>
           <name>FLASHCFG</name>
@@ -14441,6 +14440,10 @@
         <size>0x3FC</size>
         <usage>registers</usage>
       </addressBlock>
+      <interrupt>
+        <name>BOD</name>
+        <value>13</value>
+      </interrupt>
       <registers>
         <register>
           <name>SYSMEMREMAP</name>
