--- LPC845.svd.orig	2019-03-07 20:55:52.732144676 +0100
+++ LPC845.svd	2019-03-07 22:00:56.263335056 +0100
@@ -2772,7 +2772,7 @@
           <addressOffset>0</addressOffset>
           <size>32</size>
           <access>read-write</access>
-          <resetValue>0x600</resetValue>
+          <resetValue>0x000</resetValue>
           <resetMask>0x7FFF</resetMask>
           <fields>
             <field>
